src/main/scala/com/netflix/edda/mongo/MongoElector.scala

```diff
@@ -121,6 +121,7 @@ class MongoElector extends Elector {
           // if we got the update then we are leader and attempt to
           // archive the old leader record
           if (result == null) {
+            if (logger.isInfoEnabled) logger.info("Error becoming leader")
```

Comment:
The logger.info call will already check if the logger is enabled at the needed level. I think the reason the explicit check is done in some other places is to avoid the overhead of creating the string when substituting values. Even for those cases I think it would be cleaner to use the parameterization supported by the logger:

http://www.slf4j.org/faq.html#logging_performance